CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## [1.3.2] - 2021-09-20
```

Comment:
version `1.3.1` has not yet been released. You can just update with your own change there.